content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -45,8 +45,9 @@ value for this field is `600`, i.e. ten minutes.
 Once created, a CertificateSigningRequest must be approved before it can be signed.
 Depending on the signer selected, a CertificateSigningRequest may be automatically approved
 by a {{< glossary_tooltip text="controller" term_id="controller" >}}.
-Otherwise, a CertificateSigningRequest must be manually approved either via the REST API (or client-go)
-or by running `kubectl certificate approve`. Likewise, a CertificateSigningRequest may also be denied,
+Otherwise, a CertificateSigningRequest must be manually approved via the REST API - the most common
```

Comment:
the hyperlink to client-go is great but otherwise the previous sentence looked and sounded much better. Is there anything more you want to add here?